docs/docs/reference/cloud/monorepos.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Monorepos"
+description: "Monorepos support in Gatsby Cloud"
+---
+
+## Introduction
+
+Gatsby Cloud has support for monorepos projects, but since there is a plethora of dependency management tools out there some issues can occure.
+This document will walk you through the tools that are supported and how to troubleshoot some edge cases
+
+## Monorepo Support
+
+The following tables shows the level of support for each tool
+
+**Legend**
+
+| Icon | Feature Capability                       |
+| ---- | ---------------------------------------- |
+| ●    | Fully Supported                          |
+| ◐    | Somewhat Supported (support is minimal ) |
+| ○    | Not Supported                            |
+
+**Support**
+
+| Tool.                            | Level of Support | Notes                                                   |
+| -------------------------------- | ---------------- | ------------------------------------------------------- |
+| Yarn Workspaces (v1)             | ◐                |                                                         |
+| Yarn Workspaces (v2/v3 with PnP) | ◐                |                                                         |
```

Comment:
Why is it only "somewhat supported", what is missing?

Reply:
we identified a bug there and it's being worked on on the cloud side

Reply:
is V1 borked?

Reply:
nope only with pnp, I will change that